package.json

```
@@ -63,7 +62,8 @@
     ],
     "moduleNameMapper": {
       "\\.(css|scss)$": "identity-obj-proxy"
-    }
+    },
+    "coverageDirectory": "./codecov-jest/"
```

```suggestion
    "coverageDirectory": "./jest-coverage/"
```
because in the `coverage.sh` you use another naming for these folders.